gbpservice/neutron/tests/unit/nfp/configurator/lib/test_filter.py

```diff
@@ -93,7 +93,7 @@ def test_call(self):
             retval = self._make_test(self._make_lb_service_context(),
                                      'get_logical_device',
                                      pool_id=[self.pools[0]['id']])
-            self.assertEqual(retval, True)
+            self.assertTrue(retval, True)
```

Comment:
Should this change go into nfp-integration branch? This is based on liberty right?